kayak_core/src/color.rs

```diff
@@ -34,6 +34,38 @@ impl Color {
         a: 1.0,
     };
 
+    /// The white color.
+    pub const GRAY: Color = Color {
+        r: 0.5,
+        g: 0.5,
+        b: 0.5,
+        a: 1.0,
+    };
+
+    /// The red color.
+    pub const RED: Color = Color {
+        r: 1.0,
+        g: 0.0,
+        b: 0.0,
+        a: 1.0,
+    };
+
+    /// The green color.
+    pub const GREEN: Color = Color {
+        r: 0.0,
+        g: 1.0,
+        b: 0.0,
+        a: 1.0,
+    };
+
+    /// The blue color.
+    pub const BLUE: Color = Color {
+        r: 0.0,
+        g: 0.0,
+        b: 1.0,
+        a: 1.0,
+    };
+
```

Comment:
This might need to change due to your other PR (#53)

Reply:
Yeah, I think I'll just hardcode the red in this PR and once the color is merged I'll update this and any other place that uses hardcoded colors